router/xgress/options.go

```diff
@@ -46,6 +46,9 @@ type Options struct {
 	MaxCloseWait time.Duration
 
 	GetSessionTimeout time.Duration
+
+	// default timeout to use when sending requests to the controller
+	ControllerTimeout time.Duration
```

Comment:
Where is this used?

Reply:
Refactoring xgress_edge to use it. xgress/request could be refactored to use it as well at some point.

Reply:
If it's in `openziti/fabric`, shouldn't we refactor something in `fabric` to use it?

Reply:
I may need to move it. The easier thing to refactor to use it doesn't have access xgress.Options. Maybe need to make it lop level router options, though not sure those are accessible in xgress implementations.

Reply:
Refactoring something like `xgress_proxy` to use `xgress.Options` doesn't seem like it would be a big deal?